cmd/githubrunnerscalesetlistener/main.go

```diff
@@ -74,10 +74,10 @@ func main() {
 	logger := logging.NewLogger(logging.LogLevelDebug)
 	ctrl.SetLogger(logger)
 
-	c.RunnerRepository = "ting-test"
-	c.RunnerOrg = "bbq-beets"
-	c.Token = ""
-	c.RunnerScaleSetName = "test-runner-scale-set"
+	if len(c.RunnerRepository) <= 0 || len(c.RunnerOrg) <= 0 || len(c.Token) <= 0 || len(c.RunnerScaleSetName) <= 0 {
```

Comment:
Since the len of type string can never be < 0, it might be more readable like this
```suggestion
	if len(c.RunnerRepository) == 0 || len(c.RunnerOrg) == 0 || len(c.Token) == 0 || len(c.RunnerScaleSetName) == 0 {
```